Src/EventCounterCollector/EventCounterCollector.Tests/EventCounterCollector.Tests/EventCounterCollectorDiagnoticListener.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Diagnostics.Tracing;
+using System.Text;
+
+namespace EventCounterCollector.Tests
+{
+    internal class EventCounterCollectorDiagnoticListener : EventListener
```

Comment:
nit: I think you meant to call this `EventCounterCollectorDiagnosticListener` (missing an s in Diagnostic)